src/ui/shared/VolumeSlider.jsx

```
@@ -2,36 +2,51 @@ import { useId } from 'react'
+        max="1"
+        step="0.1"
+        value={clampedValue}
+        onChange={onChange}
+        aria-label={label}
+        className="sr-only"
+      />
+      <div
+        className="flex gap-1 h-8 items-end cursor-pointer group"
+        onMouseLeave={() => {}}
+        role="presentation"
       >
```

Biome (2.4.4)

[error] Static Elements should not be interactive.

(lint/a11y/noStaticElementInteractions)